### PR TITLE
fix: migration of telescope for the rested changes

### PR DESCRIPTION
**Pull request by taikiFurusyo**
This PR is for the recent PRs in CauchyE/telescope and the issue #21 .

### CODE REVIEW

File: projects/portal/src/app/views/accounts/account/account.component.html [hunk 1]
```diff
@@ -1,34 +1,29 @@
...
 </ng-template>
-
 <ng-template #exist>
   <mat-card class="mb-4" *ngIf="baseAccount">
     <mat-list>
       <mat-list-item>
         <span class="whitespace-nowrap">Sequence:</span>
         <span class="flex-auto"></span>
         <span class="ml-2 break-all">{{ baseAccount?.sequence }}</span>
+        <mat-divider [inset]="true"></mat-divider>
```
Comment (Senna46) [inline projects/portal/src/app/views/accounts/account/account.component.html:13]: What's this change?
The original looks more beautiful.

**Review by Senna46 — CHANGES_REQUESTED** [review]

File: projects/portal/src/app/views/accounts/account/account.component.html (hunk 1 above)
Comment (taikiFurusyo) [reply projects/portal/src/app/views/accounts/account/account.component.html:13]: Thank you for the advise.

Comment (taikiFurusyo) [reply projects/portal/src/app/views/accounts/account/account.component.html:13]: Thank you for the advise.

**Review by Senna46 — APPROVED** [review]
LGTM 
plz re-request

**Pull request merged**